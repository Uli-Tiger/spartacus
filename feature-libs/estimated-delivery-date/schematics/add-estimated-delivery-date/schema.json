{
  "$schema": "http://json-schema.org/schema",
  "$id": "EstimatedDeliveryDateSchematics",
  "title": "Estimated Delivery Date Schematics",
  "type": "object",
  "properties": {
    "project": {
      "type": "string",
      "description": "The name of the project.",
      "$default": {
        "$source": "projectName"
      }
    },
    "debug": {
      "description": "Display additional details during the running process.",
      "type": "boolean",
      "default": false
    },
    "lazy": {
      "type": "boolean",
      "description": "Lazy load the Estimated Delivery Date.",
      "default": true
    },
    "features": {
      "type": "array",
      "uniqueItems": true,
      "default": ["Estimated-Delivery-Date"]
    }
  },
  "required": []
}
