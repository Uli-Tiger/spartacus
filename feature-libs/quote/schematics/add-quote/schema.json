{
  "$schema": "http://json-schema.org/schema",
  "$id": "QuoteSchematics",
  "title": "Quote Schematics",
  "type": "object",
  "properties": {
    "project": {
      "type": "string",
      "description": "The name of the project.",
      "$default": {
        "$source": "projectName"
      }
    },
    "debug": {
      "description": "Display additional details during the running process.",
      "type": "boolean",
      "default": false
    },
    "lazy": {
      "type": "boolean",
      "description": "Lazy load the quote features.",
      "default": true
    },
    "features": {
      "type": "array",
      "uniqueItems": true,
      "default": ["Quote"]
    }
  },
  "required": []
}
