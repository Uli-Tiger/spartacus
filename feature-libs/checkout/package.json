{
  "name": "@spartacus/checkout",
  "version": "6.8.0",
  "description": "Checkout feature library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "checkout"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/checkout",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.6.2"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.9",
    "@angular/common": "^15.2.9",
    "@angular/core": "^15.2.9",
    "@angular/forms": "^15.2.9",
    "@angular/router": "^15.2.9",
    "@ng-select/ng-select": "^10.0.3",
    "@ngrx/store": "^15.3.0",
    "@spartacus/cart": "6.8.0",
    "@spartacus/core": "6.8.0",
    "@spartacus/order": "6.8.0",
    "@spartacus/schematics": "6.8.0",
    "@spartacus/storefront": "6.8.0",
    "@spartacus/styles": "6.8.0",
    "@spartacus/user": "6.8.0",
    "bootstrap": "^4.6.2",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
