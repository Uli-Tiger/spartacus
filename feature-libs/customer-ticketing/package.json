{
  "name": "@spartacus/customer-ticketing",
  "version": "6.5.0-1",
  "description": "Customer-Ticketing library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "customer-ticketing"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/customer-ticketing",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.6.2"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.9",
    "@angular/common": "^15.2.9",
    "@angular/core": "^15.2.9",
    "@angular/forms": "^15.2.9",
    "@angular/router": "^15.2.9",
    "@spartacus/cart": "6.5.0-1",
    "@spartacus/core": "6.5.0-1",
    "@spartacus/schematics": "6.5.0-1",
    "@spartacus/storefront": "6.5.0-1",
    "@spartacus/styles": "6.5.0-1",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
