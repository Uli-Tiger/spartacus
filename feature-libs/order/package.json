{
  "name": "@spartacus/order",
  "version": "6.8.0-1",
  "description": "Order feature library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "order"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/order",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.6.2"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.9",
    "@angular/common": "^15.2.9",
    "@angular/core": "^15.2.9",
    "@angular/forms": "^15.2.9",
    "@angular/router": "^15.2.9",
    "@ng-select/ng-select": "^10.0.3",
    "@ngrx/effects": "^15.3.0",
    "@ngrx/store": "^15.3.0",
    "@spartacus/cart": "6.8.0-1",
    "@spartacus/core": "6.8.0-1",
    "@spartacus/pdf-invoices": "6.8.0-1",
    "@spartacus/schematics": "6.8.0-1",
    "@spartacus/storefront": "6.8.0-1",
    "@spartacus/styles": "6.8.0-1",
    "@spartacus/user": "6.8.0-1",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
