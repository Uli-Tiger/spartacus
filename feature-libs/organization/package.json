{
  "name": "@spartacus/organization",
  "version": "6.6.0",
  "description": "Organization library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "organization"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/organization",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.6.2"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.9",
    "@angular/common": "^15.2.9",
    "@angular/core": "^15.2.9",
    "@angular/forms": "^15.2.9",
    "@angular/router": "^15.2.9",
    "@ng-select/ng-select": "^10.0.3",
    "@ngrx/effects": "^15.3.0",
    "@ngrx/store": "^15.3.0",
    "@spartacus/cart": "6.6.0",
    "@spartacus/core": "6.6.0",
    "@spartacus/order": "6.6.0",
    "@spartacus/schematics": "6.6.0",
    "@spartacus/storefront": "6.6.0",
    "@spartacus/styles": "6.6.0",
    "@spartacus/user": "6.6.0",
    "bootstrap": "^4.6.2",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
