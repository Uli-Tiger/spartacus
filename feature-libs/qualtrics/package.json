{
  "name": "@spartacus/qualtrics",
  "version": "6.8.0",
  "description": "Qualtrics library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "qualtrics",
    "personalized",
    "management"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/qualtrics",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.6.2"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.9",
    "@angular/common": "^15.2.9",
    "@angular/core": "^15.2.9",
    "@spartacus/core": "6.8.0",
    "@spartacus/schematics": "6.8.0",
    "@spartacus/styles": "6.8.0",
    "bootstrap": "^4.6.2",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
