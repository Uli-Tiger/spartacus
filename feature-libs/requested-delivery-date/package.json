{
  "name": "@spartacus/requested-delivery-date",
  "version": "6.3.0-1",
  "description": "Requested Delivery Date library for Spartacus",
  "keywords": [
    "spartacus",
    "framework",
    "storefront",
    "requested-delivery-date"
  ],
  "homepage": "https://github.com/SAP/spartacus",
  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/requested-delivery-date",
  "license": "Apache-2.0",
  "exports": {
    ".": {
      "sass": "./_index.scss"
    }
  },
  "scripts": {
    "build:schematics": "npm run clean:schematics && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
    "clean:schematics": "../../node_modules/.bin/rimraf --glob \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
    "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
  },
  "dependencies": {
    "tslib": "^2.4.0"
  },
  "peerDependencies": {
    "@angular-devkit/schematics": "^15.2.4",
    "@angular/common": "^15.2.4",
    "@angular/core": "^15.2.4",
    "@angular/forms": "^15.2.4",
    "@spartacus/cart": "6.3.0-1",
    "@spartacus/checkout": "6.3.0-1",
    "@spartacus/core": "6.3.0-1",
    "@spartacus/order": "6.3.0-1",
    "@spartacus/schematics": "6.3.0-1",
    "@spartacus/storefront": "6.3.0-1",
    "@spartacus/styles": "6.3.0-1",
    "rxjs": "^6.6.0"
  },
  "publishConfig": {
    "access": "public"
  },
  "schematics": "./schematics/collection.json"
}
