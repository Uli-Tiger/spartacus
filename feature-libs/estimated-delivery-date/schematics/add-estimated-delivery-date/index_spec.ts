/// <reference types="jest" />

import {
  SchematicTestRunner,
  UnitTestTree,
} from '@angular-devkit/schematics/testing';
import {
  Schema as ApplicationOptions,
  Style,
} from '@schematics/angular/application/schema';
import { Schema as WorkspaceOptions } from '@schematics/angular/workspace/schema';
import {
  CART_BASE_FEATURE_NAME,
  ESTIMATED_DELIVERY_DATE_FEATURE_NAME,
  SPARTACUS_CART_BASE,
  SPARTACUS_ESTIMATED_DELIVERY_DATE,
  SPARTACUS_SCHEMATICS,
  LibraryOptions as SpartacusEddOptions,
  SpartacusOptions,
  cartBaseWrapperModulePath,
  estimatedDeliveryDateFeatureModulePath,
  orderFeatureModulePath,
  userFeatureModulePath,
} from '@spartacus/schematics';
import * as path from 'path';
import { peerDependencies } from '../../package.json';

const collectionPath = path.join(__dirname, '../collection.json');
const scssFilePath = 'src/styles/spartacus/estimated-delivery-date.scss';

describe('Spartacus Estimated-Delivery-Date schematics: ng-add', () => {
  const schematicRunner = new SchematicTestRunner(
    SPARTACUS_ESTIMATED_DELIVERY_DATE,
    collectionPath
  );

  let appTree: UnitTestTree;

  const workspaceOptions: WorkspaceOptions = {
    name: 'workspace',
    version: '0.5.0',
  };

  const appOptions: ApplicationOptions = {
    name: 'schematics-test',
    inlineStyle: false,
    inlineTemplate: false,
    style: Style.Scss,
    skipTests: false,
    projectRoot: '',
    standalone: false,
  };

  const spartacusDefaultOptions: SpartacusOptions = {
    project: 'schematics-test',
    lazy: true,
    features: [],
  };

  const libraryNoFeaturesOptions: SpartacusEddOptions = {
    project: 'schematics-test',
    lazy: true,
    features: [],
  };

  const cartBaseFeatureOptions: SpartacusEddOptions = {
    ...libraryNoFeaturesOptions,
    features: [CART_BASE_FEATURE_NAME],
  };

  const estimatedDeliveryDateFeatureOptions: SpartacusEddOptions = {
    ...libraryNoFeaturesOptions,
    features: [ESTIMATED_DELIVERY_DATE_FEATURE_NAME],
  };

  beforeEach(async () => {
    schematicRunner.registerCollection(
      SPARTACUS_SCHEMATICS,
      '../../projects/schematics/src/collection.json'
    );
    schematicRunner.registerCollection(
      SPARTACUS_CART_BASE,
      '../feature-libs/cart/schematics/collection.json'
    );

    appTree = await schematicRunner.runExternalSchematic(
      '@schematics/angular',
      'workspace',
      workspaceOptions
    );

    appTree = await schematicRunner.runExternalSchematic(
      '@schematics/angular',
      'application',
      appOptions,
      appTree
    );

    appTree = await schematicRunner.runExternalSchematic(
      SPARTACUS_SCHEMATICS,
      'ng-add',
      { ...spartacusDefaultOptions, name: 'schematics-test' },
      appTree
    );
  });

  describe('Without features', () => {
    beforeEach(async () => {
      appTree = await schematicRunner.runSchematic(
        'ng-add',
        libraryNoFeaturesOptions,
        appTree
      );
    });

    it('should not create any of the feature modules', () => {
      expect(
        appTree.exists(estimatedDeliveryDateFeatureModulePath)
      ).toBeFalsy();
    });
  });

  describe('Estimated-Delivery-Date feature', () => {
    describe('general setup', () => {
      beforeEach(async () => {
        appTree = await schematicRunner.runSchematic(
          'ng-add',
          cartBaseFeatureOptions,
          appTree
        );

        appTree = await schematicRunner.runSchematic(
          'ng-add',
          estimatedDeliveryDateFeatureOptions,
          appTree
        );
      });

      it('should install necessary Spartacus libraries', async () => {
        const packageJson = JSON.parse(appTree.readContent('package.json'));
        let dependencies: Record<string, string> = {};
        dependencies = { ...packageJson.dependencies };
        dependencies = { ...dependencies, ...packageJson.devDependencies };

        for (const toAdd in peerDependencies) {
          if (
            !peerDependencies.hasOwnProperty(toAdd) ||
            toAdd === SPARTACUS_SCHEMATICS
          ) {
            continue;
          }
          const expectedDependency = dependencies[toAdd];
          expect(expectedDependency).toBeTruthy();
        }
      });

      it('should NOT install the required feature dependencies', async () => {
        const userFeatureModule = appTree.readContent(userFeatureModulePath);
        expect(userFeatureModule).toBeFalsy();

        const orderFeatureModule = appTree.readContent(orderFeatureModulePath);
        expect(orderFeatureModule).toBeFalsy();
      });

      it('should add the feature using the lazy loading syntax', async () => {
        const module = appTree.readContent(
          estimatedDeliveryDateFeatureModulePath
        );
        expect(module).toMatchSnapshot();

        const wrapperModule = appTree.readContent(cartBaseWrapperModulePath);
        expect(wrapperModule).toMatchSnapshot();
      });

      describe('styling', () => {
        it('should create a proper scss file', () => {
          const scssContent = appTree.readContent(scssFilePath);
          expect(scssContent).toMatchSnapshot();
        });

        it('should update angular.json', async () => {
          const content = appTree.readContent('/angular.json');
          expect(content).toMatchSnapshot();
        });
      });
    });
  });

  describe('eager loading', () => {
    beforeEach(async () => {
      appTree = await schematicRunner.runSchematic(
        'ng-add',
        { ...cartBaseFeatureOptions, lazy: false },
        appTree
      );

      appTree = await schematicRunner.runSchematic(
        'ng-add',
        { ...estimatedDeliveryDateFeatureOptions, lazy: false },
        appTree
      );
    });

    it('should import appropriate modules', async () => {
      const module = appTree.readContent(
        estimatedDeliveryDateFeatureModulePath
      );
      expect(module).toMatchSnapshot();

      expect(appTree.readContent(cartBaseWrapperModulePath)).toBeFalsy();
    });
  });
});
