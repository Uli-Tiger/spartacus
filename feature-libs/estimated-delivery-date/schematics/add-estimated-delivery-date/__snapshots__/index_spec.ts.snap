// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;

exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 2`] = `
"import { NgModule } from '@angular/core';
import { CartBaseModule } from "@spartacus/cart/base";
import { EstimatedDeliveryDateModule } from "@spartacus/estimated-delivery-date";

@NgModule({
  declarations: [],
  imports: [
    CartBaseModule,
    EstimatedDeliveryDateModule
  ]
})
export class CartBaseWrapperModule { }
"
`;

exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup styling should create a proper scss file 1`] = `
"@import "../../styles-config";
@import "@spartacus/estimated-delivery-date";
"
`;

exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup styling should update angular.json 1`] = `
"{
  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
  "version": 1,
  "newProjectRoot": "",
  "projects": {
    "schematics-test": {
      "projectType": "application",
      "schematics": {
        "@schematics/angular:component": {
          "style": "scss",
          "standalone": false
        },
        "@schematics/angular:directive": {
          "standalone": false
        },
        "@schematics/angular:pipe": {
          "standalone": false
        }
      },
      "root": "",
      "sourceRoot": "src",
      "prefix": "app",
      "architect": {
        "build": {
          "builder": "@angular-devkit/build-angular:application",
          "options": {
            "outputPath": "dist/schematics-test",
            "index": "src/index.html",
            "browser": "src/main.ts",
            "polyfills": [
              "zone.js"
            ],
            "tsConfig": "tsconfig.app.json",
            "inlineStyleLanguage": "scss",
            "assets": [
              "src/favicon.ico",
              "src/assets"
            ],
            "styles": [
              "src/styles.scss",
              "src/styles/spartacus/cart.scss",
              "src/styles/spartacus/estimated-delivery-date.scss"
            ],
            "scripts": [],
            "stylePreprocessorOptions": {
              "includePaths": [
                "node_modules/"
              ]
            }
          },
          "configurations": {
            "production": {
              "budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "500kb",
                  "maximumError": "3.5mb"
                },
                {
                  "type": "anyComponentStyle",
                  "maximumWarning": "2kb",
                  "maximumError": "4kb"
                }
              ],
              "outputHashing": "all"
            },
            "development": {
              "optimization": false,
              "extractLicenses": false,
              "sourceMap": true
            }
          },
          "defaultConfiguration": "production"
        },
        "serve": {
          "builder": "@angular-devkit/build-angular:dev-server",
          "configurations": {
            "production": {
              "buildTarget": "schematics-test:build:production"
            },
            "development": {
              "buildTarget": "schematics-test:build:development"
            }
          },
          "defaultConfiguration": "development"
        },
        "extract-i18n": {
          "builder": "@angular-devkit/build-angular:extract-i18n",
          "options": {
            "buildTarget": "schematics-test:build"
          }
        },
        "test": {
          "builder": "@angular-devkit/build-angular:karma",
          "options": {
            "polyfills": [
              "zone.js",
              "zone.js/testing"
            ],
            "tsConfig": "tsconfig.spec.json",
            "inlineStyleLanguage": "scss",
            "assets": [
              "src/favicon.ico",
              "src/assets"
            ],
            "styles": [
              "src/styles.scss",
              "src/styles/spartacus/cart.scss",
              "src/styles/spartacus/estimated-delivery-date.scss"
            ],
            "scripts": [],
            "stylePreprocessorOptions": {
              "includePaths": [
                "node_modules/"
              ]
            }
          }
        }
      }
    }
  }
}"
`;

exports[`Spartacus Estimated-Delivery-Date schematics: ng-add eager loading should import appropriate modules 1`] = `""`;
